myia/api.py

```diff
@@ -48,11 +48,6 @@ def __init__(
         pipeline=standard_pipeline,
     ):
         """Initialize a MyiaFunction."""
-        # Change this once relay becomes the default backend.
-        if use_universe and backend not in ("python", "relay"):
-            raise RuntimeError(  # pragma: no cover
-                "Universe is only supported for the relay backend."
-            )
```

Comment:
This check should probably stay as it is not for the tests but for end users.  Unless I'm missing something in this PR.